neuro_san/http_sidecar/http_sidecar.py

```diff
@@ -98,13 +102,24 @@ def make_app(self):
     def allow(self, agent_name) -> bool:
         return self.allowed_agents.get(agent_name, False)
 
+    def update_agents(self, agents: List[str]):
+        with self.lock:
+            # We assume all agents from "agents" dictionary are enabled:
+            for agent_name in agents:
+                self.allowed_agents[agent_name] = True
+            # All other agents are disabled:
+            for agent_name, _ in self.allowed_agents.items():
+                if agent_name not in agents:
+                    self.allowed_agents[agent_name] = False
+
     def build_request_data(self) -> Dict[str, Any]:
         """
         Build request data for Http handlers.
         :return: a dictionary with request data to be passed to a http handler.
         """
         return {
             "agent_policy": self,
+            "agents_updater": self,
             "port": self.port,
```

Comment:
Pass ourselves as AgentsUpdater interface implementation
to all requests handlers.
Each request handler will query the host gRPC endpoint for current state of things.